Modules/sre.h

```diff
@@ -16,12 +16,14 @@
 /* size of a code word (must be unsigned short or larger, and
    large enough to hold a UCS4 character) */
 #define SRE_CODE Py_UCS4
+
+/* SRE_MAXGROUPS is 1,073,741,823 */
+#define SRE_MAXGROUPS (INT_MAX / 2)
```

Comment:
Why is it changed? On 32-bit platform `match->mark` requires 8 bytes per group, so SRE_MAXGROUPS was `2**28-1`.